### PR TITLE
Updated Overview Page and other minor edits

### PR DESCRIPTION
**Pull request by StevenMih**
1) Added updated images to overview page
2) updated text to clarify connectors - pointing to docs
3) move the requirements to getting started
4) Added use cases

5) homepage added Discussions

@adamjshook @aweisberg @dborkar 

image

### CODE REVIEW

**Comment by StevenMih** [comment]
@aweisberg 

That second image doesn't make sense because the output of one random worker on the bottom left is SQL. The metadata API is consulted in several locations not just parsing and planning I think. Storage API is similar.

I also don't know what the data stream API is or why people who are looking at an overview page want that level of detail. That's my main reaction. Too much detail for the scope of the page. I think what people want to know is Presto has a coordinator and a worker, the coordinator parses, plans, and schedules SQL, communciates with a metastore and some storage services, and the workers communicate with storage services.

>> we updated the graphic, per your feedback - our dev agrees with you on the "data stream API" and took it out. The feedback from some users was they wanted to know what was inside the coordinator, we think this is a close approximation but open to changes 

image


-----

TPC-H isn't mentioned anymore and TPC-DS is what is mentioned?
>>> we updated the text, adding back TPC-H: 
Hive for HDFS or Object Stores (S3), MySQL, ElasticSearch, Cassandra, Kafka and more. In addition for benchmarking you can use the TPC-H or TPC-DS connectors.

-----

We dropped mention of requirements, but deploying Presto doesn't seem to mention them either 👩‍✈️
>> Yes, we moved the requirements to the getting started page where people download

File: website/static/overview.html [hunk 1]
```diff
@@ -47,77 +47,61 @@ <h1>Overview</h1>
...
 <div class="content homecontent width clearfix">
     <div class="">
-        <h1>Introduction</h1>
+        <h1>Introduction to Presto</h1>
 
         <p>
-            Presto is a distributed system that runs on a cluster of machines.
-            A full installation includes a coordinator and multiple workers.
+            Presto is a distributed system that runs on a cluster of machines. It enables 
+	analytics on large amounts of data. With Presto, access and query data in place on
```
Comment (aweisberg) [inline website/static/overview.html:54]: With Presto, access and query data in place on -> With Presto you can access and query data in place on many different data sources using ANSI SQL.

File: website/static/overview.html [hunk 2]
```diff
@@ -47,77 +47,61 @@ <h1>Overview</h1>
...
+	many data different data sources (see image below). Presto supports ANSI SQL.
+        </p>
+
+   <!--suppress CheckImageSize -- larger file for retina -->
+        <img src="static/presto-overview-2.png" width="80%" alt="Presto System Overview">
+
+	<p>
+         A full installation includes a coordinator and multiple workers.
```
Comment (aweisberg) [inline website/static/overview.html:62]: installation -> deployment

File: website/static/overview.html [hunk 3]
```diff
@@ -47,77 +47,61 @@ <h1>Overview</h1>
...
             Queries are submitted from a client such as the Presto CLI to the
             coordinator. The coordinator parses, analyzes and plans the query
             execution, then distributes the processing to the workers.
         </p>
 
         <!--suppress CheckImageSize -- larger file for retina -->
-        <img src="static/presto-overview.png" width="80%" alt="Presto Installation Overview">
-
-        <h1>Requirements</h1>
+        <img src="static/presto-architecture.png" width="80%" alt="Presto Installation Overview">
```
Comment (aweisberg) [inline website/static/overview.html:69]: What is Presto CL looker? Google doesn't know what that is.

**Comment by aweisberg** [comment]
@StevenMih Do you have time to do another pass on this?

**Comment by StevenMih** [comment]
I've made the three edits and am now opening another PR

**Pull request closed without merging**